sklearn/tests/test_common.py

```diff
@@ -377,3 +378,15 @@ def test_transformer_n_iter():
 
         if hasattr(estimator, "max_iter") and name not in external_solver:
             yield check_transformer_n_iter, name, estimator
+
+def test_get_params_invariance():
```

Comment:
Sorry, I overlooked that. We should try to unify tests as much as possible, but I guess for a regression test the current variant is fine.